common/searchattribute/defs.go

```diff
@@ -81,6 +81,14 @@ const (
 	// define a custom ScheduleId search attribute, in which case the query using the ScheduleId would operate just like
 	// any other custom search attribute.
 	ScheduleID = "ScheduleId"
+
+	// PausedEntities is a search attribute that stores the list paused entities in the workflow.
+	// Format of a single paused entity: "<entity_type>:<entity_id>:<policy_id>".
+	//  * entity_type can be activity, workflow, etc.
```

Comment:
Maybe create an enum type, and reference it here.

Reply:
done